src/main/java/com/ajou/hertz/domain/user/repository/UserProfileImageRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.ajou.hertz.domain.user.repository;
+
+import java.util.Optional;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import com.ajou.hertz.domain.user.entity.UserProfileImage;
+
+public interface UserProfileImageRepository extends JpaRepository<UserProfileImage, Long> {
+	Optional<UserProfileImage> findById(Long userId);
+}
```

Comment:
이거 기능구현 잘못된 거 같은데요?
user id로 where 조건 걸어서 조회하는거라면 이렇게 method naming 하시면 안됩니다.
참고 내용: https://docs.spring.io/spring-data/jpa/reference/jpa/query-methods.html
```suggestion
public interface UserProfileImageRepository extends JpaRepository<UserProfileImage, Long> {
	Optional<UserProfileImage> findByUser_Id(Long userId);
}
```